api/sys_seal.go

```diff
@@ -3,57 +3,91 @@ package api
 import "context"
 
 func (c *Sys) SealStatus() (*SealStatusResponse, error) {
+	return c.SealStatusWithContext(context.Background())
+}
+
+func (c *Sys) SealStatusWithContext(ctx context.Context) (*SealStatusResponse, error) {
+	ctx, cancelFunc := c.c.withConfiguredTimeout(ctx)
+	defer cancelFunc()
+
```

Comment:
```suggestion
```
This is redundant since we already call withConfiguredTimeout in sealStatusRequestWithContext helper

This applies to all other functions that call `sealStatusRequestWithContext`